src/components/reservation/ReservationModalBody.tsx

```diff
@@ -6,7 +6,6 @@ import {
   getManifestationType,
   materialIsFiction
 } from "../../core/utils/helpers/general";
-import Modal from "../../core/utils/modal";
```

Comment:
We have talked about cleaning up the commit history as a part of working with a PR. This is an example of a commit that it would be nice to have merged into the previous commit 63c71fc1b1b5449649f46c3de769dae6e6b196c6 which made the import obsolete.